qa/backwards-5.0/build.gradle

```diff
@@ -38,7 +38,7 @@ integTest {
   cluster {
     numNodes = 4
     numBwcNodes = 2
-    bwcVersion = "5.3.0-SNAPSHOT"
+    bwcVersion = "6.0.0-alpha1-SNAPSHOT" // TODO switch back to 5.3 and remove blacklist once backported
```

Comment:
yeah I know, I can merge the backport first and get a stack build published. Its just a difficult dance. We're going to have builds that fail unless it is timed perfectly. If we merge it with this change and then only revert the bwcVersion when the builds are published I think we can avoid noise in CI